pkg/virt-controller/watch/snapshot_base.go

```diff
@@ -381,6 +414,13 @@ func (ctrl *SnapshotController) handleVM(obj interface{}) {
 		for _, k := range keys {
 			ctrl.vmSnapshotQueue.Add(k)
 		}
+
+		key, err := controller.KeyFunc(vm)
+		if err != nil {
+			log.Log.Error("Failed to extract vmKey from VirtualMachine.")
+		} else {
+			ctrl.vmSnapshotStatusQueue.Add(key)
```

Comment:
Also need to add a handler for `VolumeSnapshotClass` that queues every VM whenever one is updated